debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -183,7 +183,8 @@
                                                                         OFFSET_FLUSH_INTERVAL_MS, OFFSET_COMMIT_TIMEOUT_MS,
                                                                         INTERNAL_KEY_CONVERTER_CLASS, INTERNAL_VALUE_CONVERTER_CLASS);
 
-    private static final int WAIT_FOR_COMPLETION_BEFORE_INTERRUPT = 2;
+    private static final int WAIT_FOR_COMPLETION_BEFORE_INTERRUPT_DEFAULT = 2;
+    private static final String WAIT_FOR_COMPLETION_BEFORE_INTERRUPT_PROP = "debezium.embedded.shutdown.pause.before.interrupt";
```

Comment:
This should end in "ms" and be given in milli-seconds to be consistent with similar options we have.